package.json

```diff
@@ -28,8 +28,21 @@
         "fetch": "npm-run-all --parallel fetch:*",
         "start": "npm-run-all build:sass --parallel watch:*",
         "build": "npm-run-all install:playground build:sass build:eleventy build:webpack images",
-        "lint": "eslint --ext=.js,.jsx . && markdownlint \"**/*.md\" -i node_modules",
-        "fix": "npm run lint -- --fix"
+        "minify:svg": "svgo -r -f ./",
```

Comment:
wondering how this would work? We don't actually have the svgo dep installed in the repo and we are running it via npx?

Reply:
Note: seems like svgo comes as deps via imagemin package so there is no need to install it and works out of the box

Reply:
yeah